src/node_report.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SRC_NODE_REPORT_H_
+#define SRC_NODE_REPORT_H_
+
+#include "nan.h"
+
+using v8::Isolate;
+using v8::Local;
+using v8::Message;
+using v8::Function;
+using v8::Object;
+using v8::Number;
+using v8::String;
+using v8::Value;
```

Comment:
You should really wrap this in a namespace.  `using` directives in the global namespace in header files are a perennial source of name conflicts.